examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,351 @@
+    color="yellow",
+)
+fig.show()
+
+########################################################################################
+# Using :meth:`pandas.date_range`
+# -------------------------------
+#
+# In the following example, a ``pandas.date_range`` object is used to pass date
+# data to the PyGMT figure. This object is set using the pandas method ``date_range()``.
+# This particular object contains 7 different :class:`pandas.DatetimeIndex` objects, with the
+# number being manipulated by the periods argument. Each period begins at the start
+# of a business quarter as denoted by BQS when passed to the ``periods`` parameter. The inital
+# date is the first argument that is passed to :meth:`pandas.date_range` and it marks the first
+# data in the object ``x`` that will be plotted.
+
+x = pd.date_range("2018-03-01", periods=7, freq="BQS")
+y = [4, 5, 6, 8, 6, 3, 5]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2017, 12, 31), datetime.datetime(2019, 12, 31), 0, 10],
+    frame=["WSen", "ag"],
+    x=x,
+    y=y,
+    style="i0.4c",
+    pen="1p",
+    color="purple",
+)
+fig.show()
+
+########################################################################################
+# Using :class:`xarray.DataArray`
+# ------------------------------
+#
+# In this example, instead of using a :meth:`pandas.date_range` object, ``x`` is initialized
+# as an :class:`xarray.DataArray` object. Such object provides a wrapper around traditional
+# data formats allowing this data to have varying labeled dimensions and support
+# operations that use various pieces of metadata. The following code uses a
+# :meth:`pandas.date_range` object to fill the DataArray with data, yet this is not
+# essential for the creation of a valid DataArray.
+
+x = xr.DataArray(data=pd.date_range(start="2020-01-01", periods=4, freq="Q"))
+y = [4, 7, 5, 6]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2020, 1, 1), datetime.datetime(2021, 4, 1), 0, 10],
+    frame=["WSen", "ag"],
+    x=x,
+    y=y,
+    style="n0.4c",
+    pen="1p",
+    color="red",
+)
+fig.show()
+
+###############################################################################
+# Using :class:`numpy.datetime64`
+# ------------------------------
+# In this example, instead of using a ``pd.date_range`` object, ``x`` is initialized
+# as an ``np.array`` object. Similar to ``xarray.DataArray`` this wraps the
+# dataset before passing it as a paramater. However, ``np.array`` objects use less
+# memory and allow developers to specify datatypes, and even pass multiple.
+#
+
+x = np.array(["2010-06-01", "2011-06-01T12", "2012-01-01T12:34:56"], dtype="datetime64")
+y = [2, 7, 5]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2010, 1, 1), datetime.datetime(2012, 6, 1), 0, 10],
+    frame=["WS", "ag"],
+    x=x,
+    y=y,
+    style="s0.5c",
+    pen="1p",
+    color="black",
```

Comment:
```suggestion
    color="black",
```
Please use another fill color here.